packages/studio-base/src/panels/index.ts

```diff
@@ -56,6 +56,15 @@ const builtin: PanelInfo[] = [
     thumbnail: threeDimensionalVizThumbnail,
     module: async () => await import("./ThreeDimensionalViz"),
   },
+  {
+    title: "3D (Beta)",
+    type: "3D",
+    description:
+      "Display visualization markers and models in a 3D scene using a new rendering engine.",
```

Comment:
At first glance this text is very similar to the old 3d panel (except the last few words). While it is in beta, it's probably best to get straight to the point, like:

```suggestion
      "Test our next-generation 3D panel, using a new rendering engine.",
```